build_tools/patch_linux_so.py

```diff
@@ -67,7 +67,7 @@ def add_prefix(args: argparse.Namespace):
 
         # Remove old links.
         for orig_path in orig_paths:
-            if orig_path.exists(follow_symlinks=False):
+            if orig_path.is_symlink():
```

Comment:
```suggestion
            if orig_path.exists() or orig_path.is_symlink():
```
Is closer to `orig_path.exists(follow_symlinks=False)` but checking if it is a symlink is probably sufficient (?).

Reply:
Agreed, just checking for is_symlink is fine in this case.